app/src/server/tasks/evaluateTestSetEntries.task.ts

```diff
@@ -103,11 +125,14 @@ export const evaluateTestSetEntries = defineTask<EvalKey>({
 
     const firstResultHandled =
       (firstResult.status === "IN_PROGRESS" || firstResult.status === "COMPLETE") &&
+      // backfill results if they were created without a score (3/9/2024 bugfix)
+      firstResult.score !== null &&
```

Comment:
Any eval with null comparison scores as a result of this bug should automatically fix itself once this change is merged in.